python/meep.i

```diff
@@ -571,4 +571,10 @@ extern boolean point_in_objectp(vector3 p, GEOMETRIC_OBJECT o);
             # this variable reference is needed for lazy initialization of MPI
             comm = MPI.COMM_WORLD
             master_printf('\n**\n** successfully loaded python MPI module (mpi4py)\n**\n')
+
+            if comm.Get_rank() != 0:
+                import sys
+                saved_stdout = sys.stdout
+                sys.stdout = None
```

Comment:
This might break some code that expects `stdout` to be an IO stream?   Seems better to set it to `open(os.devnull, 'w')`.

Reply:
Changed.